evm/p2p/kademlia.py

```diff
@@ -81,7 +81,7 @@ def __init__(self, pubkey, address):
     @classmethod
     def from_uri(cls, uri):
         ip, port, pubkey = host_port_pubkey_from_uri(uri)
-        return cls(pubkey, Address(ip.decode(), int(port)))
+        return cls(pubkey, Address(ip.decode(), int(port), int(port)))
```

Comment:
Just one line but this seems like it could get extracted and merged separately.

Reply:
This was left by accident, I'll remove it